### ISSUE TITLE
Update nuget packages to use `license` instead of `licenseUrl`

### ISSUE THREAD

**Issue by tsimbalar** [serilog/serilog#1265]
CI builds of *Serilog.Sinks.Console* and *Serilog.Settings.Configuration* print out a warning message [here](https://ci.appveyor.com/project/serilog/serilog-sinks-console/builds/21386397#L95) and [there](https://ci.appveyor.com/project/serilog/serilog-settings-configuration/builds/21387244#L118) : 

```
NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
```

Based on [announcement by the Nuget team](https://github.com/NuGet/Announcements/issues/32) : 
> 'licenseUrl' as package metadata is now deprecated, use 'license' instead

Based on the [Nuget documentation](https://docs.microsoft.com/en-us/nuget/reference/nuspec#license), we should therefore start changing all the Nuget package metadata of all the Serilog projects toward this new format : 

```
<license type="expression">Apache-2.0</license>
```

or in a `.csproj` file (see [docs](https://github.com/NuGet/Home/wiki/Packaging-License-within-the-nupkg-(Technical-Spec)#license-expression-pack)) : 

```
<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

This will reference the [Apache 2.0 license](https://spdx.org/licenses/Apache-2.0.html), the license that is used across the "official" Serilog repos.

Because it is a change across many repos, and does not have a high priority, I think it makes sense to apply the change little by little, as we release new versions of the sinks and other Serilog extensions.

**Comment by tsimbalar** [comment]
There was a comment in https://github.com/serilog/serilog-settings-configuration/pull/158#issuecomment-451581984 about whether we should maybe use *license file* instead (i.e. include a file containing the license as part of the nuget package, and point to that file).

I think it's better to be consistent across the Serilog repos, so I'd let @nblumhardt give his opinion, and go in that direction :)

Nick, any opinion on the topic ? Should we use : 
1. *license expression* = `Apache v2.0`
2. *license file* = our `LICENSE` file in every repo (which is a copy of the Apache v2.0 license I believe)

**Comment by nblumhardt** [comment]
Shipped in 2.9.0-dev-*
